app/views/partners/requests/_item_request.html.erb

```diff
@@ -9,7 +9,7 @@
       <%= field.number_field :quantity, label: false, step: 1, min: 1, class: 'form-control' %>
     </td>
     <td>
-      <%= remove_item_button "Remove" %>
+      <%= input_remove_button "Remove", container_selector: "tr" %>
```

Comment:
Not sure about the name... are you removing an input?

Reply:
> are you removing an input?

Not necessarily. `element_add/remove`? `html_add/remove`? With both methods you are removing/adding a chunk of HTML specified by some element.

Reply:
maybe `remove_element_button` might be better? Trying to think of synonyms (like `fragment` or `chunk`) but they all seem to have other connotations.